releasenotes/notes/52193.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: telemetry
+issue: []
+releaseNotes:
+  - |
```

Comment:
Include in the release notes the changes to the [`rootCertExpiryTimestamp`](https://github.com/istio/istio/blob/26637f432d5eb77bc6969f3647da5cd947461234/security/pkg/server/ca/monitoring.go#L58) and [`certChainExpiryTimestamp`](https://github.com/istio/istio/blob/26637f432d5eb77bc6969f3647da5cd947461234/security/pkg/server/ca/monitoring.go#L68C2-L68C26) metric so that users who expected a UNIX timestamp and a negative result to be returned can be aware